dlt/helpers/streamlit_app/menu.py

```diff
@@ -0,0 +1,42 @@
+import dlt
+import streamlit as st
+
+from dlt.common.destination.reference import WithStateSync
+from dlt.helpers.streamlit_app.utils import HERE
+from dlt.helpers.streamlit_app.widgets import logo, stat, tag, mode_selector
+from dlt.helpers.streamlit_app.blocks.load_info import last_load_info
+from dlt.pipeline.state_sync import load_pipeline_state_from_destination
+
+
+def menu(pipeline: dlt.Pipeline) -> None:
+    mode_selector()
+    logo()
+    st.page_link(f"{HERE}/dashboard.py", label="Explore data", icon="🕹️")
+    st.page_link(f"{HERE}/pages/load_info.py", label="Load info", icon="💾")
+    pipeline_state_info(pipeline)
+    last_load_info(pipeline)
+
+
+def pipeline_state_info(pipeline: dlt.Pipeline) -> None:
```

Comment:
why this is here and not in widgets or blocks?

Reply:
Moved it to blocks.